spec/bootstrap/environment_spec.rb

```diff
@@ -0,0 +1,6 @@
+# encoding: utf-8
+require "spec_helper"
+require "bootstrap/environment"
+
+describe LogStash::Environment do
+end
```

Comment:
no, I can remove it..I just thought of keeping it to encourage future tests in this module :)